replies.js

```diff
@@ -0,0 +1,271 @@
+    'Es mit mir aufzunehmen gleicht einer Odyssee.']
+  ],
+
+  [
+    'Ich werde dich richten, und es gibt kein Plädoyer!',
+    ['Das ich nicht lache! Du und welche Armee?',
+    'Jetzt werde ich dich erstechen, da hilft kein Protegé!']
+  ],
+
+  [
+    'En garde! Touché.',
+    ['Oh, das ist ein solch übles Klischee!',
+    'Deine Mutter trägt ein Toupet.']
+  ],
+
+  [
+    'Ein jeder hat vor meiner Schwertkunst kapituliert!',
+    ['Dein Geruch allein reicht aus und ich wär’ kollabiert!',
+    'Ich weiß nicht, welche meiner Eigenschaften dir am meisten imponiert.']
+  ],
+
+  [
+    'Fühl’ ich den Stahl in der Hand, bin ich in meinem Metier!',
+    ['Ich glaub’, es gibt für dich noch eine Stelle beim Varieté.',
+    'Ich laufe auf glühenden Kohlen und Barfuß im Schnee.']
+  ],
+
+  [
+    'Mein Mienenspiel zeigt dir meine Mißbilligung!',
+    ['Für dein Gesicht bekommst du ’ne Begnadigung!',
+    'Mein Antlitz zeugt von edler Abstammung!']
+  ],
+
+  // Escape from Monkey Island
+  [
+    'Memmen wie dich vernasch’ ich zum Frühstück.',
+    ['Dann ist alles klar, du bist deshalb so dick!']
+  ],
+
+  [
+    'Viele Menschen sagen, meine Kraft ist unglaublich.',
+    ['Unglaublich erbärmlich, das sag’ ab jetzt ich!']
+  ],
+
+  [
+    'Alle Welt fürchtet die Kraft meiner Faust.',
+    ['Wobei mir vor allem vor deinem Atem graust.']
+  ],
+
+  [
+    'Ich habe Muskeln an Stellen von denen du nichst ahnst.',
+    ['Zu schade, dass keiner davon in deinen Armen ist.']
+  ],
+
+  [
+    'Ich hab’ mit diesen Armen schon Kraken bezwungen.',
+    ['Und Babys wohl auch, na, der Witz ist gelungen.']
+  ],
+
+  [
+    'Ich kenne Läuse mit stärkeren Muskeln.',
+    ['Behalt sie für dich, sonst bekomm’ ich noch Pusteln.']
+  ],
+
+  [
+    'Gib auf oder ich zerquetsch’ dich wie eine lästige Mücke.',
+    ['Wenn ich mit dir fertig bin, brauchst du ’ne Krücke!']
+  ],
+
+  [
+    'Meine Großmutter hat mehr Kraft als du Wicht!',
+    ['Dafür hab’ ich in der Hand nicht die Gicht!']
+  ],
+
+  [
+    'Nach diesem Spiel trägst du den Arm in Gips.',
+    ['Das sind große Worte für ’nen Kerl ohne Grips!']
+  ],
+
+  [
+    'Hey, schau mal da drüben!',
+    ['Ja, ja, ich weiß, ein dreiköpfiger Affe!']
+  ],
+
+  [
+    'Du bist das hässlichste Wesen, das ich jemals sah',
+    ['Mit Ausnahme von deiner Frau, so viel ist klar!']
+  ],
+
+  [
+    'Ich zerreiße deine Hand in eine Million Fetzen.',
+    ['Ich wusste gar nicht, dass du so weit zählen kannst.']
+  ],
+
+  [
+    'Ich werde deine Knochen zu Brei zermalmen.',
+    ['Ich werde mich wehren, bis die Griffel dir qualmen!']
+  ],
+
+  [
+    'Sehe ich da Spuren der Angst in deinem Gesicht?',
+    ['Ha, ha! Das ist ein Lachen, du schwächlicher Wicht!']
+  ],
+
+  [
+    'Gibt es auf dieser Welt ’ne größere Memme als dich?',
+    ['Sie sitzt mir gegenüber, also was fragst du mich?']
+  ],
+]);
+
+// jscs:enable requireTrailingComma
+
+const fuzzy = FuzzySet([...insults.keys()]);
```

Comment:
What does this line do?

I read in the manual:
Read-only elements that specify the parenthesized substring matches, if included in the regular expression. The number of possible parenthesized substrings is unlimited.